client/client-multi/client-objective-c/Kaa/persistance/KaaClientPropertiesState.m

```diff
@@ -257,23 +264,23 @@ - (NSString *)refreshEndpointAccessToken {
 }
 
 - (NSData *)publicKeyAsBytes {
-    [self getOrGenerateKeyPair];
+    [self getOrGenerateKeyPair:self.isAutogeneratedKeys];
```

Comment:
There is no need to pass this as parameter since it's accessible from the method itself